src/pycookiecheat/chrome.py

```diff
@@ -66,7 +66,7 @@ def clean(decrypted: bytes) -> str:
 
 
 def chrome_decrypt(
-    encrypted_value: bytes, key: bytes, init_vector: bytes
+    encrypted_value: bytes, key: bytes, init_vector: bytes, cookie_database_version: int
```

Comment:
Do we know for sure that this is backwards compatible for at least the last 5 years or so of Chrome versions? If not, perhaps a default value that falls back to old behavior (e.g. `23` with an explanatory comment, or `None` with an extra conditional).

Reply:
Also, I know it's old school, but I still follow PEP8's 79 chars -- please wrap.